packages/flutter_tools/templates/app_shared/web/index.html.tmpl

```diff
@@ -31,74 +31,33 @@
 
   <title>{{projectName}}</title>
   <link rel="manifest" href="manifest.json">
+
+  <script>
+    // This script tag belongs to the Flutter Web build, do not touch.
```

Comment:
I'd move the comment above the `<script>` tag, and instead of "belongs...do not touch" say "the contents are auto-generated, do not edit manually".